markdown/org/docs/patterns/waralee/instructions/en.md

```diff
@@ -1,54 +1,73 @@
+Now bring the whole thing back to the outside, and sew a seam around it, about 5mm from the edge. Push it back inside, press and stitch 6mm from the edge to complete the French seam.
 
-## Step 2: Hem the sides and bottom
+As a last step, I sew the side closest to the side through all layers. This anchors the pocket to that side well.
+
+Make sure you sew the top of the pocket into the waist hem.
+
+#### Back Pockets
+
+### Step 2: Hem the sides and bottom
 
 ![Hem the sides and bottom](waralee-hem-sides-and-bottom.png)
 
-Hem the sides and bottom.
+- Hem the sides 
+- Hem the bottom.
+
+<Note>
 
 You can use whatever method works best for you. In the version I first made I just serged the sides, folded them back and top stitched them, then did the same for the bottom. I did this to keep the bulk of the fabric to a minimum and allow for a better drape. What you do should depend on the type of fabric you're using.
 
-## Step 3: Sew the two parts together at the crotch
+</Note>
+
+### Step 3: Sew the two pants parts together at the crotch
 
 ![Lay the two parts on top of one another with good sides together. Sew the crotch seam](waralee-crotch-seam-no-pockets.png)
 
-Now you get to sew the two parts together. Lay them with _good sides together_ and sew the seam along the cutout. Finish this seam and press it.
+Now you get to sew the two pants parts together.
 
-## Step 4: Prepare the four strings
+- Lay the two pants parts on top of one another with _good sides together_ and sew the seam along the cutout.  
+- _Finish_ this seam and press it.
 
-The strings are just long tubes of fabric.
+### Step 4: Prepare the four strings
 
-Take a quarter of your waist measurement. Add that number to 40cm (17”). Add your seam allowance. That is the length of each string. Cut out four strips of fabric that are that long and 5cm (2”) wide.
+The strings are just long tubes of fabric.
 
-For example, if my waist circumference is 84.6cm: 84.6cm divided by 4 is 21.2cm. 21.2cm plus 40cm is 61.2cm. 61.2cm plus my seam allowance of 1cm is 62.2cm. I will cut four strips of fabric that are 62.2cm by 5cm.
+- Take a quarter of your waist measurement. Add that number to 40cm (17”). Add your seam allowance. This is the length of each string. 
+- Cut out four strips of fabric that are that long and 5cm (2”) wide.
 
 <Note>
 
+For example, if my waist circumference is 84.6cm: 84.6cm divided by 4 is 21.2cm. 21.2cm plus 40cm is 61.2cm. 61.2cm plus my seam allowance of 1cm is 62.2cm. I will cut four strips of fabric that are 62.2cm by 5cm.
```

Comment:
Did you mean to move this into the note? It seems unrelated to the main topic ("You can lengthen the front ties").

Reply:
Just reopening this as I think it is still relevant.

Reply:
I thought I had moved that ocrrect posiition damn it git

Reply:
Actually re-reading. Yes I did but it should be a separate tip so have done that. it is an example so it can the clutter the bullet points.